Assets/MRTK/Services/InputSystem/GazeProvider.cs

```diff
@@ -151,8 +150,8 @@ public IMixedRealityInputSource GazeInputSource
         private bool delayInitialization = true;
         private Vector3 lastHeadPosition = Vector3.zero;
 
-        private Vector3? overrideHeadPosition = null;
-        private Vector3? overrideHeadForward = null;
+        public Vector3? overrideHeadPosition { get; private set; }
```

Comment:
if these are changing to properties, the capitalization should be updated to match the coding guidelines https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/contributing/coding-guidelines?view=mrtkunity-2021-05#naming-conventions

Reply:
But really...do these need to be `public`?

Reply:
I remember there being some quirk with the interface implementation requirements that necessitated that these be public for the ConeCastGazeProvider subclass, otherwise we'd have to redefine these properties again to match the normal GazeProvider's behavior.

Perhaps they can be protected instead? @ErianVazquez-Microsoft